vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue

```diff
@@ -100,42 +159,47 @@ function changeTab(val) {
   tab.value = val;
 }
 
-const FILTER_TYPE_QUERY = gql`
-  query FilterType {
-    filterType @client
-  }
-`;
+const { repos, loading } = getUserRepos(props.username, false);
+
+const { result: searchData, loading: loadingSearch } = useQuery(SEARCH_QUERY);
 
 const { result: filterTypeData, loading: loadingFilterType } =
   useQuery(FILTER_TYPE_QUERY);
 
-const { repos, loading } = getUserRepos(props.username, false);
+const { result: sortByData, loading: loadingSortBy } = useQuery(SORT_BY_QUERY);
+
+const { result: languageData, loading: loadingLanguage } =
+  useQuery(LANGUAGE_QUERY);
```

Comment:
Please add the expected result's type to `useQuery` call. You can do so like so `useQuery<T>(query)` where `T` represents the type of the result you expect